### PR TITLE
Final Submission by TEAM ALPHA Q

### PR DESCRIPTION
**Pull request by shivambhatia5432**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

[] Registration Pull request.

[x] Submission Pull request.

## Checklist ✅  

- [x] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [x] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [x] I have used the Markdown file template to add my information for the Hackathon.
- [x] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [x] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

File: content/finalist/ALPHA Q/index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+description: "Hi, We are a team of Web developers."
+Team-Quote: ""
+Application_Type: "Web Application"
+---
+
+Hey ! We are Team ALPHA Q. We are here to win and we also love web development.
+
+---
+
+# Project's Name
```
Comment (Uyadav207) [inline content/finalist/ALPHA Q/index.md]: @shivambhatia5432 , Mention Project's Name

File: content/finalist/ALPHA Q/index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+We tried to make UI simple and colourful as this App is made for catering only younger audience i.e. 6-10 years old kids.
+
+
+Idea: So we are aiming to start a positive news and media company for kids. Just imagine how does a 10-year-old kid would look at the world if he is seeing news tv channels 24x7 with his family every day. There are riots, pandemic, encounters, suicides and youtube is filled with objectionable content so a question arises do our kids really need to see all this? Yes, they do need to understand the concept of poverty, depression and pandemic but we don't need to bombard them with all the negative news and Indian parents also don't have an option these channels, newspapers and twitter are the only sources for their updates and news.
+
+So we want to solve this problem, in simple words: We want to build a positive platform for kids to learn new things and see that this world is not bad at all, good things happen every day here. Most kids can hardly read so we have also attached listen to news feature with each article and speed of the audio can also be changed.
+
+Website-Link: https://glee-hack.herokuapp.com
```
Comment (Uyadav207) [inline content/finalist/ALPHA Q/index.md]: Do not give website Link Twice

File: content/finalist/ALPHA Q/index.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+_**Role-In-Team**_  : Backend Development
+
+_**Working-Stack**_ : MERN Stack
+
+_**University/School**_ :SRM University
+
+
+
+# Member3 Name : Uditanshu Tomar
```
Comment (Uyadav207) [inline content/finalist/ALPHA Q/index.md]: mention member Name :  like   # Shivam Bhatia

**Review by Uyadav207 — CHANGES_REQUESTED** [review]
please make changes ASAP

**Comment by shivambhatia5432** [comment]
Changes requested have been implemented.

**Review by Uyadav207 — APPROVED** [review]
welcome to finals , @shivambhatia5432

**Comment by shivambhatia5432** [comment]
Thank you so much man, it has been a wonderful journey till here. Hoping for the best.

**Pull request merged**